src/main.cpp

```diff
@@ -16,7 +16,10 @@
 #include <sstream>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.